### PR TITLE
Add implementation of string searcher

### PR DESCRIPTION
**Pull request by joaoestudante**
This PR implements the requested string searcher, and includes a handy poem to search as well.

### CODE REVIEW

File: src/com/student/Main.java [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
 
+import java.io.*;
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
+        System.out.println("Usage: search <pattern> <full path to file>");
+        String command = "";
+        while (!command.equals("exit")) {
```
Comment (adamhagglund) [inline src/com/student/Main.java]: Where does exit come in? Maybe use .hasNextLine() + do the first read before the while?

**Review by adamhagglund — CHANGES_REQUESTED** [review]
See comment in code.

File: src/com/student/Main.java (hunk 1 above)
Comment (joaoestudante) [reply src/com/student/Main.java]: That makes more sense than introducing a new command, yes. Will change.

**Review by adamhagglund — APPROVED** [review]
Satisfied.

**Pull request merged**